src/Integration.Vsix.UnitTests/CFamily/MsvcDriverTest.cs

```diff
@@ -682,6 +665,27 @@ public void unsupported_std_version()
             });
         }
 
+        [TestMethod]
+        public void unsupported_option_with_argument()
```

Comment:
I'm not sure that this is the behavior that we want.
From a user's standpoint, wouldn't you rather get an exception and fix your configuration? rather than us randomly picking the last item.
What if a user did something like `/AI c:\\file.cpp a`?
I think it would be better to just tell him that we've identified multiple files and leave the fix to him.

Reply:
The user cannot do that, we are building the command and we are explicitly putting the saved file as the last element. Look at `ToCaptures`. We want to analyze the saved file and the saved file is always the last element.